content/applications/sales/sales/send_quotations/deadline.rst

```diff
@@ -7,54 +7,50 @@ encourages customers to act quickly during sales negotiations, for they might fe
 on a good deal. As well, deadlines also can also act as protection for a company in case an order
 has to be fulfilled at a price that is no longer profitable for the business.
 
-Expiration date deadlines
-=========================
+Quotation expiration
+====================
 
-On an Odoo *Sales* quotation, add an expiration date in the :guilabel:`Expiration` field located at
-the top of the quotation or sales order form.
+In Odoo *Sales*, there's the option to add an expiration date to a quotation.
 
 To add an expiration date to a quotation, navigate to :menuselection:`Sales app`, and select a
-desired quotation, or create a new one by clicking :guilabel:`Create`.
+desired quotation, or create a new one by clicking :guilabel:`New`.
 
-On the quotation form, click :guilabel:`Edit` (if modifying a pre-existing quotation), and click the
-:guilabel:`Expiration` field. Doing so reveals a drop-down calendar, in which the month and date can
-be designated as the expiration date of the quotation.
+On the quotation form, click the :guilabel:`Expiration` field to reveal a pop-up calendar. From this
+pop-up calendar, select the desired month and date as the expiration date for the quotation.
 
 .. image:: deadline/quotation-deadlines-expiration-field.png
    :align: center
-   :alt: How to configure deadlines on Odoo Sales.
+   :alt: The expiration field on a standard quotation form in Odoo Sales.
 
-Once the expiration date is selected, click :guilabel:`Save` to save all changes.
+.. tip::
+   By clicking the :guilabel:`Preview` button on a quotation, Odoo clearly displays when that
+   specific offer expires. As a reminder, the number of days is the same as those mentioned in the
+   quotation template (if a quotation template was used for the initial quotation).
```

Comment:
```suggestion
   specific offer expires. As a reminder, the expiration date is the same as those mentioned in the
   quotation template (if a quotation template was used for the initial quotation).
```
I know the quotation template lets you set the expiration date out by a specific number of days, but I was only able to get the preview to show the "This offer expires on" field with the expiration date, and not in a "in X amount of days". I am not sure if I am doing something wrong (I thought you could have it read by amount of days but I could be mistaken)

Reply:
I just double-checked, and you are totally right. It only shows the specific date, *not* the number of days. Thanks for catching this!